src/language/language.ts

```diff
@@ -0,0 +1,661 @@
+import { SQLMonarchLanguage, grafanaStandardSQLLanguage, grafanaStandardSQLLanguageConf } from '@grafana/experimental';
+
+// Language based on standard SQL, but edited according to https://docs.aws.amazon.com/redshift/latest/dg/c_SQL_commands.html
+
+export const logicalOperators = [
```

Comment:
all this is already written in the official language for Redshift: https://github.com/microsoft/monaco-editor/blob/main/src/basic-languages/redshift/redshift.ts#L34

can we use it here?